scripts/funcotator/data_sources/gnomAD/mergeVcfs_gnomadPartialOutput.json

```diff
@@ -0,0 +1,10 @@
oHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-14/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-15/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-16/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-17/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-18/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-19/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-2/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-20/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-21/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-22/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-3/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-4/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-5/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-6/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-7/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-8/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz", "gs://broad-dsde-methods/cromwell-execution-36/LiftoverVcf/41584155-e54d-4725-8775-496ed28b07d5/call-LiftoverVcfTask/shard-9/gnomad.genomes.r2.1.sites.liftoverToHg38.vcf.gz" ],
```

Comment:
You should move these files somewhere accessible to the public, right?  This bucket is private.

Reply:
The liftover file is now checked into the codebase.  I don't know which bucket is appropriate to house it.  I will reach out to comms to let them know this file exists and they can put it somewhere appropriate.

In general, these json files are preserved here to show how the liftover was run.  In the past I have had trouble verifying the provenance of data sets.  Preserving these files is one way to keep it.